modules/websocket/lws_client.cpp

```diff
@@ -34,6 +34,10 @@
 #include "core/io/stream_peer_ssl.h"
 #include "tls/mbedtls/wrapper/include/openssl/ssl.h"
 
+#if defined(MINGW_ENABLED) || defined(_MSC_VER)
+#define strncpy strncpy_s
```

Comment:
What warning was `strncpy` generating? I don't think MINGW was showing anything here.

Reply:
Oh, never mind, found it:
```
modules\websocket\lws_client.cpp(83): warning C4996: 'strncpy': This function or variable may be unsafe. Consider using strncpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
```